### ISSUE TITLE
Slack Quickstart command

### ISSUE THREAD

**Issue by brampling** [kubeshop/botkube#715]
A BotKube user needs to be presented with a quickstart dialog when they first join a Slack channel so that they can get immediate value in their environment.

The quickstart should also be accessible by running `@botkube quickstart`.

## AC

- Quickstart is shown when BotKube is installed and links to a Slack channel (first connection)
  - save state to not show it per app restart 
  - public message shown for everyone only once, right after installation for configured channels (initial config)
  - if the bot is invited to other channel, the message won't be printed
- A user runs @botkube quickstart
- The quickstart dialog shows the dialog as per the design
- The buttons run the appropriate commands

## Out of scope

- A new user joins a BotKube channel (shown only to that user)
   - Reason: BotKube can be configured for multiple channels
- Notification presets configuration

## Notes

- Unknown: whether we can write E2E tests that check if the interactive buttons work
  - if Slack Go client doesn't have such method, skip that part - test the `@BotKube quickstart` command only
- All messages are visible for everyone
- Assume #631 is resolved (merged into `main`)
- Initial focus on Slack
- Follow-up in future releases
- Ensure telemetry is in place

**Comment by lpetkov** [comment]
[Slack prototype (desktop)](https://www.figma.com/proto/iGGgUG9vn6C9WvDAFwKS9G/Onboarding?page-id=0%3A1&node-id=27%3A77&viewport=1588%2C-709%2C0.27&scaling=contain&starting-point-node-id=27%3A77)
[Slack prototype (iOS) ](https://www.figma.com/proto/iGGgUG9vn6C9WvDAFwKS9G/Onboarding?page-id=44%3A522&node-id=44%3A522&viewport=1689%2C404%2C0.85&scaling=scale-down&starting-point-node-id=44%3A523)

**Comment by mszostok** [comment]
Simplified flow:
- drop the `verbosity` for now and do the proposal first
- print the quick start only when Botkube is added to a channel
  - Currently, we post the `...and now my watch begins for cluster 'local'!` each time the BotKube is restarted, so this will require some work (maybe we can utilize `member_joined_channel`).
- A user can run `@BotKube quickstart`
- Don't print the `<User> adjusted the BotKube verbosity settings to Crictical events only.` Print instead a generic message, `The BotKube verbosity settings were changed to Crictical events only.`
  - reason: generic implementation will be complicated and can be added later. In next iteration.
- Don't show only to that user, print it publically on channel 
  - reason: generic implementation will be complicated and can be added later. In next iteration.

**Comment by mszostok** [comment]
Closed in favor of https://github.com/kubeshop/botkube/issues/683

Later we will refine the idea of the quick start and come up with a dedicated, more valuable approach.
